aeneas/src/vst/VstPrinter.v3

```diff
@@ -268,7 +268,6 @@ class VstPrinter extends VstVisitor<int, void> {
 		p.enter("MatchStmt", indent);
 		stmt.expr.accept(this, indent + 1);
 		Lists.apply(stmt.cases.list, printCases(_, indent + 2));
-		if (stmt.defcase != null) printCases(stmt.defcase, indent + 2);
```

Comment:
I think `defcase` won't be in the list in the case of `match () { } ... else { }`. So I think the right fix is to check it's been printed from above. So you should probably make the above `apply` into an explicit loop and check if the `defcase` was already printed.